h2o-py/h2o/estimators/glm.py

```diff
@@ -2781,3 +2801,39 @@ def allConstraintsPassed(model):
             return model._model_json["output"]["all_constraints_satisfied"]
         else:
             raise H2OValueError("allConstraintsPassed can only be called when there are linear constraints.")
+
+
+    def make_unrestricted_glm_model(self, dest=None):
+        """
+        Make unrestricted GLM model when control variables are defined.
+
+        Needs to be passed source model trained with control variables enabled. 
+
+        :param dest: (optional) destination key
+
+        :examples:
+
+        >>> d = h2o.import_file("http://s3.amazonaws.com/h2o-public-test-data/smalldata/prostate/prostate.csv")
```

Comment:
This example fails with:

H2OTypeError: Argument `control_variables` should be a ?list(string), got list [2, 3]